components/post_view/channel_intro_message/add_members_button/add_members_button.tsx

```diff
@@ -26,15 +27,18 @@ import './add_members_button.scss';
 
 import LoadingSpinner from 'components/widgets/loading/loading_spinner';
 
+import SetHeaderButton from '../set_header_button';
+import BoardsButton from '../boards_button';
+
 export interface AddMembersButtonProps {
     totalUsers?: number;
     usersLimit: number;
     channel: Channel;
-    setHeader?: React.ReactNode;
-    createBoard?: React.ReactNode;
+    showSetHeader: boolean;
+    showBoardsButton: boolean;
 }
 
-const AddMembersButton: React.FC<AddMembersButtonProps> = ({totalUsers, usersLimit, channel, setHeader, createBoard}: AddMembersButtonProps) => {
+const AddMembersButton: React.FC<AddMembersButtonProps> = ({totalUsers, usersLimit, channel, showSetHeader, showBoardsButton}: AddMembersButtonProps) => {
```

Comment:
I think we decided not to use `React.FC` anymore ... could you possibly remove that?